apps/gateway/test/setup.ts

```diff
@@ -0,0 +1,5 @@
+// vitest-env.d.ts
+
+import useTestDatabase from '$core/tests/useTestDatabase'
+
+useTestDatabase()
```

Comment:
Makes sense to put this here if we have to put inside describes anyway?

Reply:
you don't need to put it in describe :)

Reply:
Ah no. Ok nice

Reply:
fixed web